repository-data/application/src/main/resources/hcm-config/namespaces/hee/guidanceDocument.yaml

```diff
@@ -153,19 +161,10 @@ definitions:
             hint: ''
             /cluster.options:
               jcr:primaryType: frontend:pluginconfig
-          /relatedContent:
-            jcr:primaryType: frontend:plugin
-            caption: Related content
-            field: relatedContent
-            plugin.class: org.hippoecm.frontend.editor.plugins.field.NodeFieldPlugin
-            wicket.id: ${cluster.id}.left.item
-            hint: ''
-            /cluster.options:
-              jcr:primaryType: frontend:pluginconfig
           /rightHandBlocks:
             jcr:primaryType: frontend:plugin
             caption: Right hand objects
-            compoundList: hee:contactCardReference,hee:ctaCardReference,hee:externalLinksCardReference,hee:fileLinksCardReference,hee:QuickLinks
+            compoundList: hee:contactCardReference,hee:contentCards,hee:ctaCardReference,hee:externalLinksCardReference,hee:fileLinksCardReference,hee:QuickLinks
```

Comment:
Do we need the support for `hee:contentCards` (`Navigation cards`) content block on `Right hand objects` ?

Reply:
Probably, but will push for it to go into the enhancement ticket.

Reply:
Ok, in that case, could we remove `hee:contentCards` CompoundType `compoundList` property for now ?

Reply:
removed.